packages/contracts-bedrock/test/kontrol/proofs/interfaces/KontrolInterfaces.sol

```diff
@@ -31,9 +29,49 @@ interface ISuperchainConfig {
     function unpause() external;
 }
 
-interface IL1CrossDomainMessenger {
+interface ICrossDomainMessenger {
+    function xDomainMessageSender() external view returns (address);
+}
+
+interface IL1StandardBridge {
+    function paused() external view returns (bool);
+
+    function messenger() external view returns (ICrossDomainMessenger);
+
+    function otherBridge() external view returns (IL1StandardBridge);
+
+    function finalizeBridgeERC20(
+        address _localToken,
+        address _remoteToken,
+        address _from,
+        address _to,
+        uint256 _amount,
+        bytes calldata _extraData
+    )
+        external;
+
+    function finalizeBridgeETH(address _from, address _to, uint256 _amount, bytes calldata _extraData) external;
+}
+
+interface IL1ERC721Bridge {
     function paused() external view returns (bool);
 
+    function messenger() external view returns (ICrossDomainMessenger);
+
+    function otherBridge() external view returns (IL1StandardBridge);
+
+    function finalizeBridgeERC721(
+        address _localToken,
+        address _remoteToken,
+        address _from,
+        address _to,
+        uint256 _amount,
+        bytes calldata _extraData
+    )
+        external;
+}
+
+interface IL1CrossDomainMessenger {
```

Comment:
Do `IL1CrossDomainMessenger` and `ICrossDomainMessenger` need to be separate interfaces or can we unify them into a single interface?

Reply:
No need for them to be separate indeed, just merged them!